lib/util.js

```diff
@@ -420,14 +420,23 @@ function formatValue(ctx, value, recurseTimes, ln) {
   // Provide a hook for user-specified inspect functions.
   // Check that value is an object with an inspect function on it
   if (ctx.customInspect) {
-    const maybeCustomInspect = value[customInspectSymbol] || value.inspect;
+    let maybeCustom = value[customInspectSymbol];
+
+    if (!maybeCustom && value.inspect && value.inspect !== exports.inspect &&
```

Comment:
Isn't the `&& value.inspect` part unnecessary? The two remaining checks would cover that case, and I'd think would be faster.

Reply:
@cjihrig Indeed, that appears to be the case. I've removed it. Thanks.